crates/model/src/quote.rs

```diff
@@ -0,0 +1,126 @@
+use crate::{
```

Comment:
It is worth nothing that this code moved from the `orderbook` to the `model` crate as its part of the "API models" (along with others like order creation payload type) that we want to keep in a shared crate.